http/src/main/resources/jupyter/startup.sh

```diff
@@ -54,6 +57,22 @@ if [ "$UPDATE_WELDER" == "true" ] ; then
     docker-compose -f /etc/welder-docker-compose.yaml up -d
 fi
 
+# If a Jupyter start user script was specified, execute it now. It should already be in the docker container
+# via initialization in init-actions.sh (we explicitly do not want to recopy it from GCS on every cluster resume).
+if [ ! -z ${JUPYTER_START_USER_SCRIPT_URI} ] ; then
+  JUPYTER_START_USER_SCRIPT=`basename ${JUPYTER_START_USER_SCRIPT_URI}`
+  log 'Executing Jupyter user start script [$JUPYTER_START_USER_SCRIPT]...'
```

Comment:
I don't think the `log` function is in this file (should we add it?)

Reply:
Oops, copied here